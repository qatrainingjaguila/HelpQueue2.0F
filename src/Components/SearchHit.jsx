import React, { useImperativeHandle } from "react";
import { Card, Button } from "react-bootstrap";

const SearchHit = (props) => {
return( 
    <Card>
        <Card.Header as="h5">{props.title}</Card.Header>
        <Card.Body>
          <Card.Title>{props.author}</Card.Title>
          <Card.Text>
            <p>Description: </p>
            {props.description}
          </Card.Text>
          <Card.Text>
            <p>Solution: </p>
            {props.solution}
          </Card.Text>
        </Card.Body>
      </Card>
 );
}
export default SearchHit;