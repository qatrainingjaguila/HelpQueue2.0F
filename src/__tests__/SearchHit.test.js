import renderer from 'react-test-renderer';
import SearchHit from '../Components/SearchHit';

test("Matches searchhit", () => {
    const hit = renderer.create(<SearchHit/>).toJSON();
    expect(hit).toMatchSnapshot();
})