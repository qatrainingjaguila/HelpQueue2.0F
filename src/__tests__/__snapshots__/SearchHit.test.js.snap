// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches searchhit 1`] = `
<div
  className="card"
>
  <h5
    className="card-header"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    />
    <p
      className="card-text"
    >
      <p>
        Description: 
      </p>
    </p>
    <p
      className="card-text"
    >
      <p>
        Solution: 
      </p>
    </p>
  </div>
</div>
`;
